### DISCUSSION TITLE
Thank you for your contributions in May 2024

### DISCUSSION THREAD

**Discussion by MisRob** [learningequality/kolibri#12277] [Community]
Hello friends!

On behalf of the whole Learning Equality team, I'd like to extend our gratitude to all the contributors whose work we merged in May: @Wck-iipi, @galovdev, @EshaanAgg, @nathanaelg16, @joeysantia, and @BabyElias. Folders filtering on mobile is now much more user-friendly, a broken checkbox works again, lesson cards once again show progress, modal height adjustments are smoother, a broken tab now correctly displays itself as selected, there are new tests for components, and the developer documentation has wonderful new unit testing guidance with templates. Thanks for this all as well as to those who have work in progress. Not to forget, Google Summer of Code projects are in full swing!

If you haven't done so yet, you might be interested in checking out the Kolibri Contributors Community - an emerging group of active volunteers who regularly contribute to Learning Equality's GitHub repositories. It provides new ways to connect and acknowledge your contributions. Find out more [here](https://github.com/learningequality/kolibri/discussions/12156) and join us!

**Comment by radinamatic** [comment]
So many great improvements, thank you all! 😍
